src/app/(authPage)/forgot-password/page.tsx

```diff
@@ -1,7 +1,117 @@
-import React, { FC } from 'react';
+'use client';
+import {
+  Button,
+  FormControlLabel,
+  FormGroup,
+  Stack,
+  Switch,
+  Typography
+} from '@mui/material';
+import { red } from '@mui/material/colors';
+import TextInput from '@src/components/sharedComponents/TextInput';
+import React, { FC, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useRouter } from 'next/navigation';
+interface FormData {
+  email: string;
+}
+const schema = yup
```

Comment:
```suggestion
import { useRouter } from 'next/navigation';

interface FormData {
  email: string;
}

const schema = yup
```

Các block code không liên quan đến nhau thì cách 1 dòng